src/Benchmarks/RabbitMQ/MassTransit/MassTransitTestClient/MassTransitTestClient/Program.cs

```diff
@@ -1,26 +1,21 @@
-﻿using MassTransit;
-using MassTransitTestCommon;
-using System;
+﻿using System;
 using System.Configuration;
 using System.Diagnostics;
 using System.Threading.Tasks;
+using MassTransit;
+using MassTransitTestCommon;
 
 namespace MassTransitTestClient
 {
-    class Program
+    internal class Program
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
+            var taskCount = int.Parse(ConfigurationManager.AppSettings["NoOfThreads"]);
+
             var busControl = Bus.Factory.CreateUsingRabbitMq(cfg =>
             {
-                //Commented out settings below had no effect on client
-                //cfg.AutoDelete = true;
-                //cfg.Durable = false;
-                //cfg.SetQueueArgument("durable", false);
-
-                //Also there seems to be no way to turn off publisher confirms.
-
-                cfg.PrefetchCount = 50;
+                cfg.PrefetchCount = 100;
```

Comment:
Can PrefetchCount be set to 50? According to [this blog post](https://www.rabbitmq.com/blog/2012/04/25/rabbitmq-performance-measurements-part-2/), Prefetch counts above 50 do not produce any significant difference in the receive rate. Also, tests for other libraries have prefetch count set to 50.